docs/installation.md

```diff
@@ -21,6 +21,15 @@ And that's it! Now you should be able to use the `dem` command.
 
     The package name is axem-dem, but the command is `dem`.
 
+
+You can also install it with init-dem.sh from our repository. The installer script checks are you using the corresponding python version and install docker.io and axem-dem pip modul. Download it with the the following command:
```

Comment:
Rephrase to "The installer script checks if the corresponding python version is available and installs the docker and axem-dem pip modul.

(New line)
Download it with the following command:"

Reply:
fixed!